packages/twenty-front/src/modules/workflow/components/RightDrawerWorkflowSelectActionContent.tsx

```diff
@@ -0,0 +1,60 @@
+import { TabList } from '@/ui/layout/tab/components/TabList';
+import { MenuItem } from '@/ui/navigation/menu-item/components/MenuItem';
+import { useRightDrawerWorkflowSelectAction } from '@/workflow/hooks/useRightDrawerWorkflowSelectAction';
+import { Workflow } from '@/workflow/types/Workflow';
+import styled from '@emotion/styled';
+
+// FIXME: copy-pasted
```

Comment:
so this style could not be put in commun with others?

Reply:
This component styles the tab I used to show in the SelectAction drawer. As we stated, we don't want a tab, so this duplicated component will be removed in my next PR.